stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -366,6 +389,141 @@ public struct Unsafe${Mutable}BufferPointer<Element>
   public func makeIterator() -> UnsafeBufferPointerIterator<Element> {
     return UnsafeBufferPointerIterator(_position: _position, _end: _end)
   }
+  
+  /// Deallocates the heap block previously allocated at this buffer pointer’s 
+  /// base address. 
+  ///
+  /// This buffer pointer’s `baseAddress` must be `nil` or a pointer to a previously 
+  /// allocated memory block. If `baseAddress` is `nil`, this function does nothing.
+  /// Otherwise, the memory must not be initialized or `Pointee` must be a trivial 
+  /// type. This buffer pointer’s `count` must be equal to the allocated 
+  /// size of the heap block.
+  @_inlineable
+  public func deallocate() {
+    _position?.deallocate()
+  }
+
+% if Mutable:
+
+  /// Allocates uninitialized memory for the specified number of instances of
+  /// type `Element`.
+  ///
+  /// The resulting buffer references a region of memory that is bound to
+  /// `Element` and is `count * MemoryLayout<Element>.stride` bytes in size.
+  /// You must eventually deallocate the memory referenced by the returned
+  /// pointer.
+  /// 
+  /// When you allocate memory, always remember to deallocate once you're
+  /// finished.
```

Comment:
It's certainly important, but could we choose only one of the two sentences above? They say exactly the same thing.

Reply:
this duplication also already exists in UnsafeMutablePointer